general_docs/glossary.adoc

```diff
@@ -1,12 +1,89 @@
+An environmental effect model has a sensor-view message as input, which comes from the environment simulation, and a sensor-view message as output.
+The output is sent to the sensor model.
+
+FMI (Functional Mockup Interface)::
+Free standard that defines a container and an interface to exchange dynamic models using a combination of XML files, binaries, and C code zipped into a single file. +
++
+See also: https://fmi-standard.org/
+
+FMU (Functional Mockup Unit)::
+
+Simulation model that adheres to the FMI standard.
+There are two different kinds of FMU that both adhere to the FMI standard: Model Exchange (ME) and Co-Simulation (CS).
+An FMU is distributed in a ZIP file with the file extension _.fmu_. +
++
+See also: https://www.modelon.com/functional-mock-up-interface-fmi/
+
+Ground truth::
+Information set that describes the whole simulated environment around any simulated vehicle.
+It is based on data available to the simulation environment.
+
+Logical model::
+//TODO: input required from working group
+Examples:
+- Sensor-fusion model: Combines the output of multiple sensor models to produce data with less uncertainty
+- Fault-injection model which, contrary to a sensor-fusion model, may be used to increase uncertainties
+
+Object coordinate system::
+Local coordinate system whose origin may be identical to the center of the object's bounding box.
+If the origin is not identical to the center of the object's bounding box, the object documentation provides the actual definition.
+
+OSI (Open Simulation Interface)::
+Specification for interfaces between models and components of a distributed simulation.
+OSI has a strong focus on environmental perception of automated driving functions.
+It also specifies interfaces for modeling traffic participants.
+
+OSMP (OSI Sensor Model Packaging)::
+Specifies how models that use Open Simulation Interface (OSI) in simulation environments are packaged in accordance with the Functional Mock-up Interface 2.0 (FMI 2.0).
+
+Sensor coordinate system::
+Coordinate system for all entities that are part of sensor data.
+The origin is the mounting position of the physical sensor or a virtual mounting position, depending on the OSI message.
+
+Sensor model::
+//TODO: research definition
+Simulation of a real sensor.
+Sensor models consume sensor-view messages and produce sensor-data messages.
+Sensor-model output does not represent raw data but detected features or classified objects.
+
+Sensor view::
+Information set that is derived from ground truth and used as input to sensor models.
+
+Traffic participant::
+Represents one of the following:
+- Living being.
+- Means of transportation for living beings or goods.
+- Any other movable object that travels the road network.
+
+A traffic participant is an element of the simulated world and can change its state during simulation time, for example, its position and orientation.
+
+Traffic participant model::
+//TODO: input needed
```

Comment:
The traffic participant model makes decisions how to change position, orientation and other states of the dedicated moving object.
To do this, it reacts on input given by SensorView and TrafficCommand resulting in TrafficUpdate as output.